packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -82,6 +82,8 @@ function createLanguageClient(context: ExtensionContext): LanguageClient {
       fileEvents: [
         workspace.createFileSystemWatcher("**/manifest.json"),
         workspace.createFileSystemWatcher("**/ui5.yaml"),
+        workspace.createFileSystemWatcher("**/*.xml"),
```

Comment:
maybe this can also be centralized in the `context` to just provide a map of file extensions next to the `watcher.js` in which the reaction to changes could also be moved to ?